merino/curated_recommendations/provider.py

```diff
@@ -116,14 +119,17 @@ async def fetch(
                 # interest vector. Data science work shows that using the topics as features
                 # is effective as a first pass at personalization.
                 # https://mozilla-hub.atlassian.net/wiki/x/FoV5Ww
-                features={item.topic.value: 1.0} if item.topic else {},
+                features={f"t_{item.topic.value}": 1.0} if item.topic else {},
             )
             for rank, item in enumerate(corpus_items)
         ]
 
         sections_feeds = None
         general_feed = []
-        if self.is_sections_experiment(request, surface_id):
+        is_sections_experiment = self.is_sections_experiment(request, surface_id)
+        inferred_p13n_enabled = is_sections_experiment and request.inferredInterests
```

Comment:
This would avoid the if-else condition when building the response object:
```suggestion
        inferred_local_model = None
        if is_sections_experiment and request.inferredInterests:
            inferred_local_model = self.local_model_backend.get(surface_id)
```